web-console/src/views/segments-view/segments-view.tsx

```diff
@@ -221,7 +221,7 @@ export class SegmentsView extends React.PureComponent<SegmentsViewProps, Segment
           queryParts = compact([
             `SELECT`,
             `  ("start" || '/' || "end") AS "interval",`,
-            `  "segment_id", "datasource", "start", "end", "size", "version", "partition_num", "num_replicas", "num_rows", "is_published", "is_available", "is_realtime", "is_overshadowed", "payload"`,
+            `  "segment_id", "datasource", "start", "end", "size", "version", "partition_num", "num_replicas", "num_rows", "is_published", "is_available", "is_realtime", "is_overshadowed"`,
```

Comment:
I don't know TypeScript much, but shouldn't we add the additional fields to this query ?

Reply:
Hi @surekhasaharan, I think the content in this popover already includes these fields. 
<img width="1670" alt="image" src="https://user-images.githubusercontent.com/44718283/85969961-b56d7200-b9fb-11ea-9e19-435a6e52f000.png">

Reply:
Yeah, it looks like this information is fetched outside of system table from ```/druid/coordinator/v1/metadata/datasources/${datasourceId}/segments/${segmentId}```